### ISSUE TITLE
Create simple neural net for v1 synthetic data set

### ISSUE THREAD

**Issue by rockladyeagles** [vgcagle/DARC#12]
Create the simplest possible neural net in Keras/TensorFlow to take a 38-word sequence of word embeddings (in whatever form you and @harmonyrose decide) and predict "positive" or "negative" as to the presence of an A_supplies_B relation in the text.

In case you care (you probably don't) exactly two of the ≤38 embeddings will be @harmonyrose's "special" embeddings representing `__NE_FROM__` and `__NE_TO__`. The labels (positive/negative) will be present in @rockladyeagles's synthetic data set (#8) and will be passed through @rpersing's and @harmonyrose's code to yours.

**Comment by rockladyeagles** [comment]
code is mostly complete, but waiting on labeled/vectorEmbedded data set from @harmonyrose
